packages/rum-core/src/domain/view/viewMetrics/trackScrollMetrics.ts

```diff
@@ -67,3 +88,39 @@ export function computeScrollValues() {
     scrollTop,
   }
 }
+
+export function createScrollValuesObservable(
+  configuration: RumConfiguration,
+  throttleDuration = THROTTLE_SCROLL_DURATION
+): Observable<ScrollValues> {
+  const observable = new Observable<ScrollValues>(() => {
+    function notify() {
+      observable.notify(computeScrollValues())
+    }
+
+    if (window.ResizeObserver) {
+      const throttledNotify = throttle(notify, throttleDuration, {
+        leading: false,
+        trailing: true,
+      })
+
+      const observerTarget = document.scrollingElement || document.documentElement
+      const resizeObserver = new ResizeObserver(monitor(throttledNotify.throttled))
+      resizeObserver.observe(observerTarget)
+      const eventListener = addEventListener(configuration, window, DOM_EVENT.SCROLL, throttledNotify.throttled, {
+        passive: true,
+      })
+
+      return () => {
+        throttledNotify.cancel()
+        resizeObserver.unobserve(observerTarget)
+        eventListener.stop()
+      }
+    }
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏unit tests are successful even if we remove all this section, could we add some tests on that?

Reply:
I missed the fallback intent, let's discard this comment